add_me_to_your_PATH/Dockerfile.template

```diff
@@ -116,27 +149,23 @@ RUN --mount=type=secret,id=github_token \
 
 FROM base as project
 
+ENV JULIA_PROJECT /JuliaProject
+
 # copy over artifacts generated during the `precompile-image` stage
-COPY --from=precompile-image /JuliaProject/ /JuliaProject/
+COPY --from=precompile-image /JuliaProject/ ${JULIA_PROJECT}/
 COPY --from=precompile-image /root/.julia /root/.julia
 COPY --from=precompile-image /usr/local/julia/lib/julia/sys.* /usr/local/julia/lib/julia/
 
-# contains github token, you may want to remove this file
-# but then adding private deps during julia_pod sessions will not work
-COPY --from=precompile-image /root/.gitconfig /root/.gitconfig
```

Comment:
yikes, so we've been leaking github credentials into our images with previous versions??

Reply:
This problem was addressed back in #60. This comment is definitely misleading because as of the version of the Dockerfile before this PR all this was doing was copy over the `.gitconfig` which contained a reference to the github-token-helper with no actual credentials being present.

I'll point out that this step is completely pointless as we're inheriting the `/root/.gitconfig` from the `base` stage already and the `precompile-image` stage never modifies this specific file.